client/src/components/EpisodeSearch.js

```diff
@@ -3,9 +3,10 @@ import styled from "styled-components/macro";
 import PropTypes from "prop-types";
 
 const InputField = styled.input`
-  width: 280px;
+  width: 265px;
```

Comment:
Why do you use px here instead of % and use px for larger viewports.
Try researching media queries

Reply:
Thanks for your hint! I set up a new issue to make the whole content responsive for bigger screens at a later stage 👍